alerts/openshift-virtualization-operator/HCOMisconfiguredDescheduler.md

```diff
@@ -0,0 +1,70 @@
+# HCOMisconfiguredDescheduler
+
+## Meaning
+
+A Descheduler is a OpenShift Container Platform application that causes the
```

Comment:
-> **an** OpenShift...

Also probably lower-case case d and italics for _descheduler_